src/lazy/text/raw/v1_1/reader.rs

```diff
@@ -422,34 +504,36 @@ impl<'top> LazyRawSequence<'top, TextEncoding_1_1> for LazyRawTextSExp_1_1<'top>
     }
 
     fn iter(&self) -> Self::Iterator {
-        // Make an iterator over the input bytes that follow the initial `(`; account for
-        // a leading field name and/or annotations.
-        let open_paren_index =
-            self.value.matched.encoded_value.data_offset() - self.value.matched.input.offset();
-        RawTextSExpIterator_1_1::new(self.value.matched.input.slice_to_end(open_paren_index + 1))
+        let MatchedValue::List(child_exprs) = self.value.matched.encoded_value.matched() else {
+            unreachable!("list contained a matched value of the wrong type")
+        };
+        RawTextSequenceCacheIterator_1_1::new(child_exprs)
     }
 
     fn as_value(&self) -> LazyRawTextValue_1_1<'top> {
         self.value.matched.into()
     }
 }
 
-impl<'top> Iterator for RawTextSExpIterator_1_1<'top> {
+impl<'top> Iterator for RawTextListIterator_1_1<'top> {
```

Comment:
Did `impl<'top> Iterator for RawTextListIterator_1_1` and `impl<'top> Iterator for RawSExpListIterator_1_1` get swapped for any particular reason?

Reply:
I think this is a misleading diff that came from cutting the `RawTextSExpIterator` and pasting it closer to the definition of `struct LazyRawTextSExp_1_1`.